src/components/dashboard/Dashboard.js

```diff
@@ -399,6 +398,8 @@ const Dashboard = props => {
     } catch (e) {
       log.error('withdraw failed:', e.code, e.message, e)
       showErrorDialog(e.message)
+    } finally {
+      props.navigation.setParams({ paymentCode: undefined, code: undefined, reason: undefined })
```

Comment:
this is code handling paymentcode only. only payment code need to be set.
reason? there's no such param

Reply:
@sirpy I deleted the code param as it is not required here. The reason param appears with paymentCode. If you will try to create the send link you will see two parameters: paymentCode and reason. Here is the example of send link: 
http://localhost:3000?paymentCode=a2f71cdc42e82285cbfa&reason=test

Reply:
we move all parameters into base64 code. so it will always be only one param

Reply:
@AlexeyKosinski ok. got it. Removed. See new commit